### PR TITLE
Comments can be added to posts

### PR DESCRIPTION
**Pull request by i-hardy**
Not yet able to edit or delete comments

### CODE REVIEW

File: spec/controllers/comments_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    end
+
+    it "responds with 302" do
+      post :create, params: { post_id: Post.find_by(message: "hello").id, comment: { text: "Hello, world!" } }
+      expect(response).to have_http_status(302)
+    end
+
+    it "creates a comment" do
+      post :create, params: { post_id: Post.find_by(message: "hello").id, comment: { text: "Hello, world!" } }
+      expect(Comment.find_by(text: "Hello, world!")).to be
```
Comment (dbugsy) [inline spec/controllers/comments_controller_spec.rb]: Unfinished line?

File: spec/features/users_can_post_comments.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.feature "Comments", type: :feature do
+  scenario "Can submit comments on a post" do
+    visit "/posts"
+    click_link "Comments"
+    fill_in "Text", with: "This post sucks"
+    click_button "Create comment"
+    expect(page).to have_content("This post sucks")
+  end
```
Comment (dbugsy) [inline spec/features/users_can_post_comments.rb]: nice!

File: spec/helpers/comments_helper_spec.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+# Specs in this file have access to a helper object that includes
+# the CommentsHelper. For example:
+#
+# describe CommentsHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
```
Comment (dbugsy) [inline spec/helpers/comments_helper_spec.rb]: Unused, can delete this

File: spec/models/comment_spec.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  it { is_expected.to be }
```
Comment (dbugsy) [inline spec/models/comment_spec.rb]: expected to be what?

File: app/controllers/comments_controller.rb [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+      redirect_to post_path(@post)
+    else
+      render 'posts/show'
+    end
+  end
+
+  private
+  def comment_params
+    params.require(:comment).permit(:text)
+  end
```
Comment (dbugsy) [inline app/controllers/comments_controller.rb:15]: good use of permit

File: app/controllers/comments_controller.rb [hunk 6]
```diff
@@ -0,0 +1,16 @@
+class CommentsController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.create(comment_params)
+    if @comment.save
```
Comment (dbugsy) [inline app/controllers/comments_controller.rb]: `create` will also save. better to use `.new`.

**Review by dbugsy — CHANGES_REQUESTED** [review]

**Comment by i-hardy** [comment]
Changes made and testing bug fixed, please review @dbugsy

**Review by dbugsy — APPROVED** [review]

**Comment by dbugsy** [comment]
:shipit: :shipit: 

You'll need to resolve conflicts in order to merge - you have admin, so this should be possible. Nice work!

**Review by petrakh — APPROVED** [review]
:shipit:

**Review by langphil — APPROVED** [review]
:shipit:

**Pull request merged**